api/lib/application/certification-center-memberships/index.js

```diff
@@ -2,18 +2,41 @@ import { certificationCenterMembershipController } from './certification-center-
 import { securityPreHandlers } from '../security-pre-handlers.js';
 
 const register = async function (server) {
+  const certifRoutes = [
+    {
+      method: 'PATCH',
+      path: '/api/certification-centers/{certificationCenterId}/certification-center-memberships/{id}',
+      config: {
+        handler: certificationCenterMembershipController.updateFromPixCertif,
+        pre: [
+          {
+            method: (request, h) =>
+              securityPreHandlers.adminMemberHasAtLeastOneAccessOf([
+                securityPreHandlers.checkUserIsAdminOfCertificationCenter,
+              ])(request, h),
```

Comment:
```suggestion
            method: securityPreHandlers.checkUserIsAdminOfCertificationCenter,
```

Le prehandler `adminMemberHasAtLeastOneAccessOf` n'est utile que lorsque l'on doit vérifier qu'au moins une règle est respectée.

Reply:
Cela a été corrigé avec un commit de fixup 690e5b781986591e0d03d7d446d9d3f428e56ac6